app/server/api/credit_transfer_api.py

```diff
@@ -298,7 +298,7 @@ def post(self, credit_transfer_id):
                         send_user=g.user,
                         receive_user=recipient_user,
                         uuid=uuid,
-                        transfer_subtype='DISBURSEMENT')
+                        transfer_subtype=TransferSubTypeEnum.DISBURSEMENT)
```

Comment:
Are you sure that will work @enjeyw ? might need to be `TransferSubTypeEnum.DISBURSEMENT.value`?

Reply:
I think that's what was causing trouble in the first place - `TransferSubTypeEnum.DISBURSEMENT.value` is a string, so invites string comparison, whereas u can just check equality for most things on `TransferSubTypeEnum.DISBURSEMENT`